### DISCUSSION TITLE
How to change the cefsharp browser canvas html hash for fingerprinting?

### DISCUSSION THREAD

**Discussion by wangf205x** [cefsharp/CefSharp#3972] [General]
![image](https://user-images.githubusercontent.com/3354655/150147385-6d0d3736-a3c3-4e07-8057-d70efca5c13e.png)
The fingerprints of the cefsharp browser are the same, even on different computers. Want to know how to change the fingerprint of the cefsharp browser.

**Comment by amaitland** [comment]
I'd suggest moving the conversation over to [CEF Forum](https://magpcss.org/ceforum/index.php) (post a link back here for reference). If the behaviour differs from chromium then changes in CEF are likely required. Someone there may have investigated a workaround also. 

CefSharp is just one of many chromium embedded framework(CEF) wrappers.
